### PR TITLE
BST model tests

### PR DESCRIPTION
**Pull request by EricMichaelSmith**
**Patch description**
- Adding unit tests for the BST zoo models
- Updating the `testing.eval_model` docstring, which looks a little out of date

**Testing steps**
(Running CI checks)

### CODE REVIEW

**Comment by emilydinan** [comment]
how long is this test?

**Comment by EricMichaelSmith** [comment]
> how long is this test?

Just timed it - it's a little less than 9 min on my devfair. It helps that we're only evaluating the first 1000 BST examples for each model

File: tests/tasks/test_blended_skill_talk.py [hunk 1]
```diff
@@ -402,5 +402,31 @@ def test_safe_personas(self):
...
+            'multi_task': 0.8086,
+            'multi_task_bst_tuned': 0.8457,
+        }
+        for model_name, desired_accuracy in model_names_to_desired_accuracies.items():
+            valid, _ = testing_utils.eval_model(
+                opt={
+                    'batchsize': 64,
+                    'model_file': f'zoo:blended_skill_talk/{model_name}/model',
+                    'task': 'blended_skill_talk',
+                    'num_examples': 1000,
```
Comment (klshuster) [inline tests/tasks/test_blended_skill_talk.py:424]: i would even suggest dropping this down to like 100 examples (cut from 9 minutes to under a min)

Comment (stephenroller) [reply tests/tasks/test_blended_skill_talk.py:424]: Also why not a multiple of 64 ;)

Comment (stephenroller) [reply tests/tasks/test_blended_skill_talk.py:424]: Also heads up, the GPUs in CircleCI are like 5 generations behind our devfairs. If it's 9 minutes on your devfair, it's going to be like 30 on the CI machines.

File: tests/tasks/test_blended_skill_talk.py [hunk 2]
```diff
@@ -402,5 +402,31 @@ def test_safe_personas(self):
             self.assertEqual(len(personas), count)
 
 
+class TestBlendedSkillTalkModels(unittest.TestCase):
```
Comment (stephenroller) [inline tests/tasks/test_blended_skill_talk.py:405]: This should go in the night/gpu tests folder, not the tasks folder.

Comment (EricMichaelSmith) [reply tests/tasks/test_blended_skill_talk.py]: Makes sense - moved

File: tests/tasks/test_blended_skill_talk.py (hunk 1 above)
Comment (EricMichaelSmith) [reply tests/tasks/test_blended_skill_talk.py]: Yes - reducing this to 64

**Comment by EricMichaelSmith** [comment]
Notes on CI checks:
- @stephenroller mypy check currently fails because of error when installing dependencies: `FileNotFoundError: [Errno 2] No such file or directory: '/opt/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/PyYAML-5.3.1-py3.7-linux-x86_64.egg'`
- long_gpu_tests succeeded upon re-running: https://app.circleci.com/pipelines/github/facebookresearch/ParlAI/5956/workflows/72155208-7580-4061-b740-e8550fe4d1d9/jobs/44615/parallel-runs/3?filterBy=ALL

**Comment by klshuster** [comment]
I've been getting that error as well on other PRs fwiw

**Pull request merged**